lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -43,8 +43,16 @@
     }
   }
 
+  &:active {
+    background-color: transparent;
```

Comment:
I suspect this change is incorrect but I'm waiting to hear back from @alexalyons126 [here](https://sonatype.slack.com/archives/CJUF9G7T2/p1631128816043100?thread_ts=1631114504.042400&cid=CJUF9G7T2)

Reply:
> I suspect this change is incorrect but I'm waiting to hear back from @alexalyons126 [here](https://sonatype.slack.com/archives/CJUF9G7T2/p1631128816043100?thread_ts=1631114504.042400&cid=CJUF9G7T2)

Thanks! I did reach out regarding this topic. Apparently the Dropdown button behaviour is following the same pattern as the form fields and not button so it remains transparent. I will further confirm this with @alexalyons126 and get back to you.